components/Admin/Add/index.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+  <BaseModal
+    :show="modalShow"
+    :loading="loading"
+    :is-form-completed="isFormCompleted"
+    label-right-btn="Tambahkan"
+    title="Tambah - Administrator Baru"
+    @submit="onSubmit"
+    @close="onModalClose"
+  >
+    <form v-if="modalShow" class="form-add-admin">
+      <div class="grid grid-cols-4">
+        <div class="col-span-1">
+          <div
+            :class="{
+              'form-add-admin__image': true,
+              'form-add-admin__image--attached': isAttached
+            }"
+          >
+            <jds-icon size="14px" name="user" />
+          </div>
+        </div>
+        <div class="col-span-3">
+          <div class="form-add-admin__title">
+            Upload Foto Profile
+          </div>
+          <div>
+            <div>
+              Ukuran maksimal file adalah 1 Mb.
+            </div>
+            <div>
+              File yang didukung adalah .jpg dan .png
+            </div>
+          </div>
+          <div class="form-add-admin__button">
+            <button class="form-add-admin__button-btn" type="button" @click="$refs.file.click()">
+              Tambah File
+              <jds-icon class="ml-2" size="12px" name="plus-bold" />
+            </button>
+            <input
+              ref="file"
+              type="file"
+              hidden="true"
+              accept="image/png, image/jpeg, image/svg+xml"
+              @change="onFileChange"
+            >
+          </div>
+          <div v-if="fileImage">
+            Filename: {{ fileImage.get('file').name }}
+          </div>
+          <div v-else-if="uploadFileErrorMessage" class="text-red-700">
+            {{ uploadFileErrorMessage }}
+          </div>
+          <div v-else>
+            Belum ada file terpilih.
+          </div>
+        </div>
+      </div>
+      <div class="form-add-admin__form-group">
+        <jds-input-text
+          v-model="form.name"
+          name="nama"
+          label="Nama Administrator"
+          placeholder="Masukkan nama Administrator"
+        />
+      </div>
+      <div class="form-add-admin__form-group">
+        <jds-input-text
+          v-model="form.email"
+          name="email"
+          label="Email"
+          placeholder="Masukkan alamat email"
+        />
+      </div>
+      <div class="form-add-admin__form-group">
+        <BaseInputText
+          v-model="form.password"
+          name="password"
+          label="Kata Sandi"
+          placeholder="Masukkan kata sandi"
+          type="password"
+        />
+        <!-- <jds-input-text
```

Comment:
Minor,
Hello kak @yoslie, is this code still use?

Reply:
it's not used thanks @doohanas , fixed [here](https://github.com/jabardigitalservice/dedi-cms/pull/27/commits/5c9cfc24749bfa64728bfcc01f0a892ec21d68d2)